webapps/api/backend/views/vdisks.py

```diff
@@ -130,21 +130,13 @@ def get_children(self, vdisk, hints):
         Returns a list of vDisk guid(s) of children of a given vDisk
         :param vdisk: Vdisk to get the children from
         :type vdisk: VDisk
-        :param hints: Hits provided by the load decorator
+        :param hints: Hints provided by the load decorator
         :type hints: dict
+        :return: Guids of the child vDisks
+        :rtype: list
         """
-        children_vdisk_guids = []
-        children_vdisks = []
-        if vdisk.is_vtemplate is False:
-            raise HttpNotAcceptableException(error_description='vDisk is not a vTemplate',
-                                             error='impossible_request')
-        for cdisk in vdisk.child_vdisks:
-            if cdisk.guid not in children_vdisk_guids:
-                children_vdisk_guids.append(cdisk.guid)
-                if hints['full'] is True:
-                    # Only load full object is required
-                    children_vdisks.append(cdisk)
-        return children_vdisks if hints['full'] is True else children_vdisk_guids
+        _ = hints
```

Comment:
If you don't use the parameter, just remove it, the ```@load``` decorator will do the rest.